metaproc-connector/src/com/axonivy/connector/metaproc/auth/AuthTokenFeature.java

```diff
@@ -53,13 +53,13 @@ public void filter(ClientRequestContext context) throws IOException {
               .accept(MediaType.APPLICATION_JSON).post(entity)) {
         var node = response.readEntity(ObjectNode.class);
         TOKEN = node.get("loginResult").get("token").asText();
-        TOKEN_EXPIRED_TIME = System.currentTimeMillis() + getExpiredTime();
+        TOKEN_EXPIRED_TIME = System.currentTimeMillis() + getExpiredDuration();
         context.getHeaders().add(TOKEN_HEADER, TOKEN);
       }
     }
   }
   
-  private long getExpiredTime() {
+  private long getExpiredDuration() {
 	  return Long.valueOf(Ivy.var().get("metaproc.TokenExpiredDuration")) * 60 * 1000;
```

Comment:
time conversions are hard to read and maintain: it's better to read java's time helpers: 
```
var seconds = Long.valueOf(Ivy.var().get("metaproc.TokenExpiredDuration"));
return TimeUnit.SECONDS.toMillis(seconds);
```